helm/revproxy/gen3.nginx.conf/local-grip.conf

```diff
@@ -0,0 +1,31 @@
+location /grip/graphql/synthea {
```

Comment:
why is synthea hardcoded here?

Reply:
yeah that needs to be edited. WIP currently

Reply:
okay will leave this convo open until you get to it